lang/machine.md

```diff
@@ -162,6 +196,16 @@ impl<M: Memory> Machine<M> {
 
         self.thread_manager.threads.mutate_at(active_thread, f)
     } 
+
+    fn fn_from_addr(&self, addr: mem::Address) -> NdResult<Function> {
```

Comment:
This can probably be just `Result`? (also see https://github.com/RalfJung/minirust/pull/98)